src/domain/permissions.js

```diff
@@ -115,5 +119,7 @@ export default {
   OWN_ROOMS,
   DELETE_FOREIGN_ROOMS,
   AUTORIZE_ROOMS_RESOURCES,
-  JOIN_PRIVATE_ROOMS
+  JOIN_ROOMS,
+  CREATE_DOCUMENT_COMMENT,
+  MANAGE_DOCUMENT_COMMENT
```

Comment:
In combination with db "entitites" we mostly used the plural so far (like "manage-storage-plans" etc.). I'm not insisting, just preferring :-)

Reply:
Nicely spotting, I of course aligned the namings, thanks!